app/controllers/feedbacks_controller.rb

```diff
@@ -32,4 +32,13 @@ def load_support_languages
     @support_languages = Language.where(support_available: true).order(:name)
   end
 
+  private
+
+  def feedback_params
+    params.require(:feedback).permit(
+      :comment, :email, :summary, :user_agent, :category, :approved,
```

Comment:
Looking over the model code, `approved` is where we save the result of our spam check -- so it's definitely not something we want to let the user submit. It seems like it might be helpful to check the column names against the form fields that are in the view, although it probably is easier for our reviewers to spot things that _shouldn't_ be allowed than things that are missing.